api/src/main/java/org/openmrs/Condition.java

```diff
@@ -340,34 +340,34 @@ public boolean equals(Object o) {
 			return false;
 		}
 		
-		Condition condition = (Condition) o;
+		Condition conditionObj = (Condition) o;
```

Comment:
Why rename to conditionObj?

Reply:
On line 55, we have another object named condition and this local local object was hiding the global object as identified by sonar. So I just renamed it to conditionObj

Thank you!

Reply:
Should I create a seperate PR for renaming condition?